examples/browser-webrtc/src/lib.rs

```diff
@@ -46,6 +51,19 @@ pub async fn run(libp2p_endpoint: String) -> Result<(), JsError> {
                 tracing::info!("Ping successful: RTT: {rtt:?}, from {peer}");
                 body.append_p(&format!("RTT: {rtt:?} at {}", Date::new_0().to_string()))?;
             }
+            SwarmEvent::ConnectionClosed {
+                cause: Some(cause), ..
+            } => {
+                tracing::info!("Swarm event: {:?}", cause);
+
+                if let libp2p::swarm::ConnectionError::KeepAliveTimeout = cause {
+                    body.append_p("All done with pinging! ")?;
+
+                    break;
+                } else {
```

Comment:
This `else` is not needed because you `break` before (which I think is good style!)

Reply:
Ah, good eyes! I'll fix this.